sp800-63a/sec3_definitions.md

```diff
@@ -6,19 +6,25 @@
 
 _This section is informative._
 
-There is a wide variety of terms used in the area of authentication. While the definitions of many terms are consistent with earlier versions of SP 800-63, some have changed in this revision. Since there is no single, consistent definition of many of these terms, careful attention to how the terms are defined here is warranted.
+>**MG: I started editing it all, but then switched to only editing terms that are only in 63A. Hopefully caught them all.**
+
+There is a wide variety of terms used in the area of digital identity. While the definitions of many terms are consistent with earlier versions of SP 800-63, some have changed in this revision. Since there is no single, consistent definition of many of these terms, careful attention to how the terms are defined here is warranted.
 
 The definitions in this section are primarily those that are referenced in this document. Refer to the other documents in the SP 800-63 document family for additional definitions and abbreviations specific to their content.
 
 #### Address of Record
 The validated and verified location (physical or digital) where an individual can receive communications using approved mechanisms.
 
+>**MG: flagging on the same email address concern from before. Will folks read this as making email a valid address of record akin to physical mail. I know normative language is clearer, but there could be confusion.**
```

Comment:
email address can be an address of record.  Depending on the use case we allow it and our very clear when it must be posted.  Frankly, I don't think address of record gets you a whole lot other than delivery of enrollment code to potentially be used later.  Binding of attributes to the user is done in many other ways now - whereas in the past this was the only way.